vm/systemSmartContracts/auction.go

```diff
@@ -177,6 +174,10 @@ func (s *stakingAuctionSC) Execute(args *vmcommon.ContractCallInput) vmcommon.Re
 		return s.unBondTokens(args)
 	case "updateStakingV2":
 		return s.updateStakingV2(args)
+	case "unBondTokensWithNodes":
```

Comment:
unitTests are needed for the 2 new functions

Reply:
unit tests are needed for everything. they will come in separate PRs.